modules/uncertainty/R/ensemble.R

```diff
@@ -409,7 +411,9 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
       do.call(my.write.config, args = list( defaults = defaults, 
                                             trait.values = lapply(samples$parameters$samples, function(x, n) { x[n, , drop=FALSE] }, n=i), # this is the params
                                             settings = settings, 
-                                            run.id = run.id
+                                            run.id = run.id,
+					    obs_time = time,
+					    update_phenology = update_phenology
```

Comment:
If you're adding obs_time and update_phenology in this way, you're going to need to go through EVERY SINGLE write.configs function for every model in PEcAn and add these two arguments to their write.configs. Please find an alternative solution.